Agent/AgentWar3/UnitRefList/Ability/AbilityInterfaced/CAbilityHero.h

```diff
@@ -16,31 +16,31 @@ class CAbilityHero : CAbilityInterfaced {
     int field_80;
     int field_84;
     int field_88;
-    int experience;
-    int skillPoints;
-    int strength; // assigned with the SetHeroStr native
```

Comment:
Can you confirm this?

Reply:
- Experience is confirmed.
- heroLevel is in fact "unallocatedSkillPoints"
- heroStartingStrength is the base strength before the hero recive any levels (a level 10 and 1 hero of the same type has the same stat on the same attribute). 
This value does not appear to increase per level but might be modified by SetHeroStr and modifying this value will increase/decrease the Hero Strength accordingly.

Reply:
Interesting about the hero strength. Maybe it should be `baseStrength`? Since it appears to be the strength before adding any effects or levels. And since it can be modified, `starting` might be a bit misleading.